anushajain19/README.md

```diff
@@ -0,0 +1,27 @@
+ #100DaysOfCode
+
+**Anusha Jain**
+
+*Second year Electrical Engineering, MNIT Jaipur*
+
+---
+
+## Goals
+
+**Competitive Programming**
+
+- () solve most of competitive programming problems.
+
+**Learning Goals**
+
+- () Learn C++ STL
+- () complete Data Structures to switch into dynamic programming concept
+- () To finish with java and android tutorials.
```

Comment:
It would be good if you would also mention the specific resource from where you wish to complete the tutorials for java and android, though that is optional.

> complete Data Structures to switch into dynamic programming concept

I did not really understand the above line, is it something like `complete data structures and clear dp concepts` kind of
Still you are free to write your own goals. Just use checkbox from markdown and you are good to go!

Reply:
Boss please take a view now

Reply:
You are making changes to the description of the commit, not the `readme` file. See if you can figure it out. Don't worry, it takes some time to get acquainted with the git workflow.

You need to go to your  fork, open your `readme` file and edit it.
If you want more help, you can message me on telegram or I can do it for you.

Reply:
oh yes 
sorry
i am doing it there